collect_app/src/main/res/xml/server_preferences.xml

```diff
@@ -8,5 +8,6 @@
         android:entryValues="@array/protocol_entry_values"
         android:key="protocol"
         android:title="@string/type"
+        android:enabled="false"
```

Comment:
I was thinking that we should only disable this for ODK server projects, but I realize that we don't want people to accidentally switch their project from Google Drive to ODK and then not be able to switch back.